components/Footer/index.tsx

```diff
@@ -8,20 +8,20 @@ function Footer() {
 
     return (
         <>
-            <footer>
-                <div className="footer-links">
-                    <div className="dmpui-frontend-container dmpui-frontend">
-                        <div className="footer-links-grid">
-                            <div className="footer-links-logo">
+            <footer className={styles.footerWrapper}>
```

Comment:
Switched to using CSS Modules for the Footer, since its styles were overriding some page styles.